roles/hosted_engine_setup/tasks/add_engine_as_ansible_host.yml

```diff
@@ -7,6 +7,7 @@
     become: false
     ansible.builtin.command: whoami
     register: username_on_host
+    changed_when: true
```

Comment:
Shouldn't it be `false`?
From some reading I did about `changed_when`, all the read-only `shell` & `command` tasks should declare `changed_when: false` and the other might even need to check the return status to be sure whether the task eventually modified something or not.
And `whoami` looks like very much read-only.
Am I right? Since I'm also fixing same kind of failures.
Same comment might be relevant for other places as well.

Reply:
AFAICT, it doesn't really matter, but yes, `changed_when: false` make more sense.